### ISSUE TITLE
arXiv harvesting is fully moved to Labs

### ISSUE THREAD

**Issue by jalavik** [inspirehep/inspire-next#405]
- [x] Enable selection of records from arXiv `math` and `astro-ph` on Labs using Holding Pen
- [ ] Add automatic rejection based on prediction score and number of CORE keywords detected
- [ ] Reorganize arXiv harvesting workflow to improved structure
- [ ] Enable harvesting of all non-core arXiv categories on Labs
- [ ] Turn off legacy arXiv harvesting and enable harvesting of all arXiv categories on Labs

**Comment by kaplun** [comment]
@jalavik when we migrate QA to Labs, does it mean we can move arXiv harvesting to Labs?

**Comment by jalavik** [comment]
@kaplun almost! There is no automatic merging yet on the Labs harvesting like todays simplified merging on legacy. Plus the automatic acceptance of CORE record categories need to be added. Both fairly simple to port. But it shouldn't be a blocker for deployment.

**Comment by kaplun** [comment]
@jalavik since this is waiting for automatic merging, is this postponed to when @mihaibivol work is complete? Is it for AB Meeting?

**Comment by jalavik** [comment]
@kaplun Yeah, post AB if we deploy Labs on v3, we can see if the merger is already integrated and it can be turned off on production.

**Comment by kaplun** [comment]
I guess you mean "on" :smile:

**Comment by jalavik** [comment]
I mean that arXiv harvesting turns *off* on production, but fully *on* on labs
